integration_test_suite/integration_tests/src/test/java/io/prometheus/jmx/test/Abstract_IT.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2022-2023 The Prometheus jmx_exporter Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.prometheus.jmx.test;
+
+import com.github.dockerjava.api.model.Ulimit;
+import org.antublue.test.engine.api.Parameter;
+import org.antublue.test.engine.api.ParameterMap;
+import org.antublue.test.engine.api.TestEngine;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Abstract_IT {
+
+    public static final String DOCKER_IMAGE_NAME = "dockerImageName";
+    public static final String IS_JAVA_6 = "isJava6";
+    public static final String MODE = "mode";
+
+    /**
+     * Method to get the list of Docker image names
+     *
+     * @return the return value
+     */
+    @TestEngine.ParameterSupplier
+    protected static Stream<Parameter> parameters() {
+        List<Parameter> parameters = new ArrayList<>();
+
+        DockerImageNames
+                .names()
+                .forEach(dockerImageName -> {
+                    for (Mode mode : Mode.VALUES) {
```

Comment:
There's no need for an explicit `VALUES` variable. Enums have a built-in `values()` method, so you could write `Mode.values()`.

Reply:
Removed.